src/presentation/http/router/note.test.ts

```diff
@@ -167,6 +175,58 @@ describe('Note API', () => {
       expect(response?.json()).toStrictEqual({ message: 'Note not found' });
     });
 
+    test('Returns public note by public id with false canEdit flag, when user is not authorized', async () => {
```

Comment:
I think the code status should be added to the description, it will make the test more understandable